### ISSUE TITLE
No keyboard shortcuts recognized, no image resized

### ISSUE THREAD

**Issue by kbitr** [AlexeyAB/Yolo_mark#23]
Hey, when using the tool no keyboard shortcuts, except the arrow keys. I'm using an english keyboard. Furthermore, the image will not be resized and so the program window is bigger than my screen.

**Comment by adonese** [comment]
I had the same issue (running on Ubuntu 16.04).

**Comment by AlexeyAB** [comment]
@kbitr @adonese Hi,

Can you put this line
```
if (pressed_key >= 0) std::cout << "pressed_key = " << (int)pressed_key << std::endl;
```
here: https://github.com/AlexeyAB/Yolo_mark/blob/d0f23d83dbf540ba3d9c8657df6509df1bd7ac29/main.cpp#L605

Then recompile and run.
Then press shortcuts buttons that aren't recognized, and write me their codes and corresponds pressed buttons?

**Comment by AlexeyAB** [comment]
Also what OpenCV version do you use?

**Comment by adonese** [comment]
I use OpenCV 3.4. When I uncomment that line, I got this `pressed_key = -1`, for any of the shortcuts mentioned on the program (c, n, space, etc)

**Comment by AlexeyAB** [comment]
@adonese Thanks. Have you this part of code?
https://github.com/AlexeyAB/Yolo_mark/blob/d0f23d83dbf540ba3d9c8657df6509df1bd7ac29/main.cpp#L557-L561

**Comment by AlexeyAB** [comment]
Thank you!
Yes, please write codes for: 0, 1, ... 9. 
Or just for 0 and 9 if the values are sequential.

**Comment by adonese** [comment]
Yes, I do. I have also tried to modify it, to only use the OpenCV 3 part. Didn't work either.

For 0 to 9
0: pressed_key = 1048624
9: pressed_key = 1048633

Yes, they are sequential.

**Comment by AlexeyAB** [comment]
@adonese I fixed it. Try to update your code and recompile.

**Comment by adonese** [comment]
Man that's really awesome! Thank you so much!

For new comers, if you'd like to use the new update, use
```shell
$ git stash # in case you made local changes
$ git pull
$ cmake .
$ make
```

**Comment by adonese** [comment]
also, is there anyway to fix the issue of the window. It cannot be resized, hence you cannot mark images of sizes greater than 416 (I assume).

**Comment by AlexeyAB** [comment]
I added ability to resize the window, try to update the code.

**Comment by adonese** [comment]
That's just perfect. Thank you so much Alex.
